src/util.rs

```diff
@@ -267,8 +267,17 @@ pub mod git {
                 "remote authentication required but none available",
             ))
         });
+
+        // Creating a proxy option to, eventually, address the usage of Sheldon
+        // behind a http proxy. Using the [auto()] function to try to
+        // auto-detect the proxy from the git configuration.
+        let mut proxy_opts = git2::ProxyOptions::new();
+        proxy_opts.auto();
+
         let mut opts = FetchOptions::new();
         opts.remote_callbacks(rcb);
+        // adds proxy_opts to opts
```

Comment:
I don't feel this comment is necessary

Reply:
Yes, my only intent was to mark my modification but this comment is unnecessary. I removed it.